components/Lists/lists.html

```diff
@@ -0,0 +1,170 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Lists</title>
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap"
+    />
+    <link
+      rel="stylesheet"
+      href="https://pro.fontawesome.com/releases/v6.0.0-beta3/css/all.css"
+    />
+    <link rel="stylesheet" href="../../css/winsome.css" />
+  </head>
+  <body>
+    <div class="grid-container">
+      <header class="header-bar">
+        <div class="header-item">
+          <div class="header-name">WINSOME-UI</div>
+        </div>
+        <div class="header-item">
+          <div class="header-icons">
+            <a
+              href="https://github.com/AnjaliDeshwani"
+              target="_blank"
+              class="header-link"
+            >
+              <i class="fa-brands fa-github"></i>
+            </a>
+          </div>
+
+          <div class="header-icons">
+            <a
+              href="https://twitter.com/anjali_deshwani"
+              target="_blank"
+              class="header-link"
+            >
+              <i class="fa-brands fa-twitter"></i>
+            </a>
+          </div>
+          <div class="header-icons">
+            <a
+              href="https://www.linkedin.com/in/anjalideshwani/"
+              target="_blank"
+              class="header-link"
+            >
+              <i class="fa-brands fa-linkedin-in"></i>
+            </a>
+          </div>
+        </div>
+      </header>
+      <div class="side-bar">
+        <nav class="nav-links">
+          <a href="/index.html" class="nav-link-item">Installation</a>
+          <a href="/components/Avatar/avatar.html" class="nav-link-item">
+            Avatar
+          </a>
+          <a href="/components/Alert/alert.html" class="nav-link-item">
+            Alert
+          </a>
+          <a href="/components/Badge/badge.html" class="nav-link-item">
+            Badge
+          </a>
+          <a href="/components/Button/button.html" class="nav-link-item">
+            Button
+          </a>
+          <a href="/components/Card/card.html" class="nav-link-item">Card</a>
+          <a href="/components/Image/image.html" class="nav-link-item active">
```

Comment:
Give the "active" class to lists rather than image component

Reply:
Thanks Tarandeep for pointing it out, I somehow missed it. I have updated it now.